components/scrollable-container/README.md

```diff
@@ -27,7 +27,7 @@ The gameface-scrollable-container component comes with UMD and CJS builds.
 * add the gameface-scrollable-container component to your html:
 
 ~~~~{.html}
-<gameface-scrollable-container class="gameface-scrollable-container-component"></gameface-scrollable-container>
+<gameface-scrollable-container class="scrollable-container-component"></gameface-scrollable-container>
```

Comment:
```suggestion
<gameface-scrollable-container class="guic-scrollable-container-component"></gameface-scrollable-container>
```

Reply:
There is a **guic-scrollable-container**`-wrapper` in the style.css, not **guic-scrollable-container**`-component`.
The class name is not strictly a component one, seems unused and just for an example. I have removed `gameface-` because in the last to the second section `## Automatically showing and resizing the scrollbar`, the name was without `-gameface` and because in the **demo.html** -> it is again without `gameface-`.

In my head there are a few actions:

- Remove the class completely from the demo.html and the document since it is not used for styling or related to any functional code (left as an example only).
- Revert the change to the original `gameface-scrollable-container-component` and change it in the `## Automatically showing and resizing the scrollbar` and in demo.html;
- Leave it as I've changed it.

What do you think?